providers/closecrm/types.go

```diff
@@ -0,0 +1,40 @@
+package closecrm
+
+import (
+	"time"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/handy"
+)
+
+type SearchParams struct {
+	ObjectName string
+	Fields     handy.StringSet
+	Since      time.Time
+	NextPage   common.NextPageToken
+	Filters    Filters
+}
+
+type Filters struct {
+	FilterQueries FilterQueries       `json:"query"`
```

Comment:
We can keep the field name as `Query` to match json schema required by provider.